src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/sdl_pending_resumption_handler.h

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
this should be renamed like waypoints as well

Reply:
@iCollin renamed in https://github.com/smartdevicelink/sdl_core/pull/3475/commits/8913095240e15b52211f29fb6af06a4bd573155a